### PR TITLE
Skip creating images atm

### PR DESCRIPTION
**Pull request by arilivigni**
- Skip generating images since it causes issues
- Also add stage message to checkLatestImage

Once this is split into libs for each stage
this will make more sense

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -738,7 +738,9 @@ def rsyncResults(stage) {
     }
 }
 
-def checkLastImage() {
+def checkLastImage(stage) {
```
Comment (scoheb) [inline Jenkinsfile]: Why not return true/false instead of touching a file?

Comment (arilivigni) [inline Jenkinsfile]: I just took what was there before that touched a file.  In the shell step how do you return that to the method being called.

Comment (scoheb) [inline Jenkinsfile]: Here is a snippet of how to do it.
https://paste.fedoraproject.org/paste/iAtLDDDTMg6POXx3vvvO1A
maybe we can tackle it as part of a refactoring PR if you prefer.

Comment (arilivigni) [inline Jenkinsfile]: I like it.  Sounds like it can be done as the massive refactoring. 
Libraries , sanitizing messages, etc.

**Comment by scoheb** [comment]
+1

**Pull request merged**